plugins/modules/authorized_key.py

```diff
@@ -28,8 +28,7 @@
     required: true
   path:
     description:
-      - Alternate path to the authorized_keys file.
-      - When unset, this value defaults to I(~/.ssh/authorized_keys).
+      - It's not necessary to set this key. The default value should be enough most of the time. 
```

Comment:
Please change that to `Most of the time. it's not necessary to set this key. The default value should just work.`. I'm afraid the previous sentence was not clear enough.